docs/scos/dev/back-end-development/zed-ui-tables/create-and-configure-zed-tables.md

```diff
@@ -131,23 +131,39 @@ $config->setDefaultSortDirection(
 
 ### Configure strict search
 
-The default search option in Back Office data tables searches for anything that contains the specified sub strings. This default search makes use of the SQL logical operator ‘LIKE’ in combination with ‘LOWER’ for comparison. It may result in performance issues on larger tables due to indexes not being used.
+The default search option in Back Office data tables searches for anything that contains the specified substrings. This default search makes use of the SQL logical operator ‘LIKE’ in combination with ‘LOWER’ for comparison. It may result in performance issues on larger tables due to indexes not being used.
 
-In order to solve the performance issues, strict search mode has been introduced and can be used on all backoffice data tables which extend the `AbstractTable` class.
+In order to solve the performance issues, a strict search mode has been introduced and can be used on all backoffice data tables which extend the `AbstractTable` class.
```

Comment:
> In order to solve the performance issues


We've already mentioned performance improvements below. (Also, this is not the only reason why we need to search per column)

Please get rid of this message.